src/mode/edifact.js

```diff
@@ -3,10 +3,12 @@
 var oop = require("../lib/oop");
 var TextMode = require("./text").Mode;
 var EdifactHighlightRules = require("./edifact_highlight_rules").EdifactHighlightRules;
+var CstyleBehaviour = require("../mode/behaviour/cstyle").CstyleBehaviour;
 
 var Mode = function() {
    
     this.HighlightRules = EdifactHighlightRules;
+    this.$behaviour = new CstyleBehaviour();
```

Comment:
Do we still need these instead of relying on default?

Reply:
We could rely on default

Reply:
Replaced all `new CstyleBehaviour()` to default ones